tests/python/pants_test/scm/test_git.py

```diff
@@ -67,7 +67,7 @@ def setUp(self):
       self.initial_rev = subprocess.check_output(['git', 'rev-parse', 'HEAD']).strip()
       subprocess.check_call(['git', 'tag', 'first'])
       subprocess.check_call(['git', 'push', '--tags', 'depot', 'master'])
-      subprocess.check_call(['git', 'branch', '--set-upstream', 'master', 'depot/master'])
+      subprocess.check_call(['git', 'branch', '--set-upstream-to', 'depot/master'])
```

Comment:
Please rebase against master to get rid of this change.

Reply:
Done.